src/react-native/react-native-wizard.ts

```diff
@@ -374,3 +374,20 @@ async function patchAndroidFiles(config: RNCliSetupConfigContent) {
     chalk.green(`Android ${chalk.cyan('app/build.gradle')} saved.`),
   );
 }
+
+async function confirmPodInstall(): Promise<boolean> {
+  return traceStep('confirm-pod-install', async () => {
+    const continueWithPodInstall = await abortIfCancelled(
+      clack.select({
+        message:'Do you want to run `pod install` now?',
```

Comment:
l: Should we hint something like "This might take several minutes"? 

Feel free to disregard if this is this doesn't make sense, I obviously lack a lot of context here

Reply:
Yes, makes sense, we have a similar message at the spinner for the pods install.